nestoria-project/src/app/modules/property-search-page/property-search.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="nav">
+  <div>
+    <h2>Property Cross</h2>
+  </div>
+  <div>
+    <app-button [id]="'button-favor'" label="Faves" click="navigateToFavor()"></app-button>
```

Comment:
Such things like 'button-favor' should be in const/enum/class-property in component or file with constants

Reply:
created enum